src/main/kotlin/it/pagopa/wallet/scheduler/jobs/paymentwallet/OnboardedPaymentWalletJob.kt

```diff
@@ -95,11 +95,12 @@ class OnboardedPaymentWalletJob(
             }
             .flatMap { cdcEventDispatcherService.dispatchEvent(it) }
             .collectList()
-            .map {
-                it.last().timestamp // <- this is the wallet creation date and can be used to save
-                // it to
-                // Redis as latest processed chunk
-                // TODO: add redis checkpoint save here with the latest saved wallet date
+            .flatMap {
+                redisResumePolicyService.saveResumeTimestamp(
+                    id(),
+                    Instant.parse(it.last().timestamp)
+                )
+                return@flatMap Mono.just(it.last().timestamp)
```

Comment:
Since saveResumeTimestamp is a synchronous blocking method (same goes for getResumeTimestamp method), what if we follow the best practice suggested by the reactor [documentation](https://projectreactor.io/docs/core/snapshot/reference/faq.html#faq.wrap-blocking)?

Reply:
@pietro-tota refactor done in [0ee1b2b](https://github.com/pagopa/pagopa-payment-wallet-scheduler-service/pull/10/commits/0ee1b2b7ed90910f57278b49a7096c05866a23a1). Let me know what do you think about it